daemon/api_system_volumes.go

```diff
@@ -141,8 +142,7 @@ func postSystemVolumesActionReplaceRecoveryKey(c *Command, req *systemVolumesAct
 		return BadRequest("cannot replace recovery key: %v", err)
```

Comment:
this code needs to do error conversion correctly to produce a 409 conflict error if there's a conflict, we also need a test for that